scripts/covid19indiaORG/README.md

```diff
@@ -0,0 +1,63 @@
+
+# Importing covid19india.org Data Into Data Commons
+
+Author: @edumorlom
+
+## Table of Contents
+
+1. [About the Dataset](#about-the-dataset)
+1. [Import Artifacts](#import-artifacts)
+1. [Generating Artifacts](#generating-artifacts)
+
+## About the Dataset
+
+### Download URL
+
+The import APIs can be found in ./Config.py.
+Each API returns the time series data for each Indian State along with the State's districts.
+
+### Overview
+
+This dataset shows how Covid-19 has impacted States and Districts in India.
+#### Active Cases
```

Comment:
Does cumulative active = simple count of currently active?

Reply:
Sorry, didn't mean to put cumulative in front of active.
Fixed.